manifests/nagios/nrpe_check_total_procs_lax.pp

```diff
@@ -3,7 +3,7 @@
   Integer $critical = 200,
   Integer $warning = 150,
 ) {
-  if is_hash($facts) and has_key($facts, 'cosmos') and ('frontend_server' in $facts['cosmos']['host_roles']) {
+  if 'cosmos' in $facts and ('frontend_server' in $facts['cosmos']['host_roles']) {
```

Comment:
Here we are removing two operations but only replacing with one: has_key() seems reasonable to replace with "if in" syntax, but "is_hash()" is just removed. Is this on purpose? Can this lead to doing "if in" on some other variable type like a string?

Reply:
Added a check to make sure it is a hash like it was before, but with new syntax.